spec/lib/api/login-spec.js

```diff
@@ -54,10 +54,12 @@ describe('Http.Api.Login', function () {
         return [
             dihelper.simpleWrapper(require('swagger-express-mw'), 'swagger'),
             dihelper.simpleWrapper(ws.Server, 'WebSocketServer'),
+            dihelper.simpleWrapper({}, 'TaskGraph.TaskGraph'),
+            dihelper.simpleWrapper({}, 'TaskGraph.Store'),
             dihelper.simpleWrapper({}, 'Task.Services.OBM'),
             dihelper.simpleWrapper({}, 'ipmi-obm-service'),
-            dihelper.requireWrapper('rimraf', 'rimraf'),
-            dihelper.requireWrapper('os-tmpdir', 'osTmpdir'),
+            dihelper.requireWrapper('rimraf', 'rimraf', undefined, __dirname),
```

Comment:
Since `rimraf` is actually an on-core dependency, the default search path for the `requireWrapper` call searches in the wrong place by default when any npm links have been made with on-core. This bug was fixed in a couple other places in https://github.com/RackHD/on-http/pull/97, but has been re-introduced with the `login-spec.js` file and I think I was just the first to run it with npm link and notice it. `__dirname` specifies the right relative search path, `undefined` is the slot in the function signature for extra injector overrides or context I think, which we don't want here.